.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
-## What happened 
+## What happened 👀
 
 Describe the big picture of your changes here to communicate to the team why we should accept this pull request.
 
-## Insight
+## Insight 📝
 
-Describe in details how to test the changes. Referenced documentation are welcome as well.
+Describe in detail how to test the changes. Referenced documentation is welcome as well.
```

Comment:
As you made some changes to the git templates, could you also open a PR to fix them on [this repository](https://github.com/nimblehq/git-template/) as well?

Reply:
@hoangmirs I give you not 1 but 2 PR :D 
A PR in the [git-template](https://github.com/nimblehq/git-template/pull/20) repo.
Another PR in the [github-actions-workflows](https://github.com/nimblehq/github-actions-workflows/pull/5) repo. ;-)